service/controller/v18/key/key.go

```diff
@@ -29,16 +29,19 @@ const (
 	// ProbeHost host for liveness probe.
 	ProbeHost = "127.0.0.1"
 	// InitialDelaySeconds is InitialDelaySeconds param in liveness probe config
-	InitialDelaySeconds = 60
+	InitialDelaySeconds = 180
 	// TimeoutSeconds is TimeoutSeconds param in liveness probe config
-	TimeoutSeconds = 3
+	TimeoutSeconds = 5
 	// PeriodSeconds is PeriodSeconds param in liveness probe config
-	PeriodSeconds = 20
+	PeriodSeconds = 30
 	// FailureThreshold is FailureThreshold param in liveness probe config
 	FailureThreshold = 4
 	// SuccessThreshold is SuccessThreshold param in liveness probe config
 	SuccessThreshold = 1
 
+	// enable k8s-kvm-health check for k8s api
```

Comment:
Should be `Enable k8s-kvm-health check for k8s api.` not sure why we are so inconsistent in this file :man_shrugging: